charts/apisix-dashboard/values.yaml

```diff
@@ -86,9 +86,17 @@ service:
 
 ingress:
   enabled: false
+  # kubernetes ingress attribe ingressClassName,default to "apisix" 
+  # Kubernetes 1.18+ support ingressClassName attribe 
+  className: "apisix"
```

Comment:
We should comment it by default

Reply:
why? is become kubernetes version not upgrade!?

Reply:
Yes, we need to maintain compatibility.

Reply:
ok, please reaudit